CHANGELOG.md

```diff
@@ -225,6 +225,8 @@ See the [release announcement](https://godotengine.org/article/godot-3-3-has-arr
 
 #### Porting
 
+- **All platforms:** [`OpenSimplexNoise` is now guaranteed to give consistent results across platforms](https://github.com/godotengine/godot/issues/47211)
```

Comment:
This should be in the Core section rather than Porting, as this isn't OS-specific. You can also remove the **All platforms:** header.

Reply:
I was in doubt because it's not really core either -- OpenSimplexNoise is a module. But you know best!